internal/integration/cli/image.go

```diff
@@ -35,6 +35,13 @@ func (suite *ImageSuite) TestDefault() {
 	)
 }
 
+// TestSourceBundle verifies source-bundle Talos list of images.
+func (suite *ImageSuite) TestSourceBundle() {
+	suite.RunCLI([]string{"image", "source-bundle", "v1.11.2"},
+		base.StdoutShouldMatch(regexp.MustCompile(regexp.QuoteMeta("ghcr.io/siderolabs/talosctl-all:v1.11.2"))),
```

Comment:
I wonder if we could match on the full output/more items here, as it should be stable?